core/jvm/src/main/scala-2.11-2.12/zio/metrics/jvm/JvmMetricsVersionSpecific.scala

```diff
@@ -0,0 +1,11 @@
+package zio.metrics.jvm
+
+import scala.collection.JavaConverters._
+
+trait JvmMetricsVersionSpecific {
```

Comment:
Minor thing but in other places we have used silencer to silence the deprecation warnings and avoid having version specific files for this.